substrate/frame/assets-freezer/Cargo.toml

```
@@ -16,46 +16,31 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { workspace = true }
-frame-benchmarking = { optional = true, workspace = true }
-frame-support = { workspace = true }
-frame-system = { workspace = true }
+frame = { workspace = true, features = ["experimental", "runtime"] }
```

We are removing the `experimental` feature flag as in #7177 to avoid spreading that feature to the whole sdk.
```suggestion
frame = { workspace = true, features = ["runtime"] }
```

---

Done for all open PRs